addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/internal/api/EvohomeApiClientV2.java

```diff
@@ -156,6 +156,9 @@ public void update() {
 
     @Override
     public ControlSystem[] getControlSystems() {
+        if(locationsStatus == null){
+            update();
```

Comment:
Shall we leave the update calls out of the getters and let it just update in the refresh task? That way we can control how often the website is contacted